utils/scripts/install-el8.sh

```diff
@@ -45,22 +45,37 @@ dnf --nodocs install \
     Lmod \
     lz4-devel \
     make \
+    meson \
     ndctl \
+    ninja-build \
     numactl \
     numactl-devel \
     openmpi-devel \
     openssl-devel \
     patch \
     patchelf \
+    python3-defusedxml \
     python3-devel \
+    python3-distro \
+    python3-junit_xml \
     python3-pip \
+    python3-pyxattr \
+    python3-pyyaml \
+    python3-scons \
+    python3-tabulate \
     sg3_utils \
     sudo \
     systemd \
     valgrind-devel \
     which \
     yasm
 
+: ${REPO_FILE_URL:=}
```

Comment:
(lint) This default assignment may cause DoS due to globbing. Quote it. [SC2223]